lib/events.mli

```diff
@@ -0,0 +1,263 @@
+(**
+
+This module provides both {{!section-highlevel}high-level} and
+{{!section-lowlevel}low-level} interfaces to the real-time events API.
+
+ *)
+
+(** {1:highlevel High-Level Interface}
+
+ *)
+
+(** The {!module-Message} module represents the content of ["message"] events
+returned by Zulip's real-time events API. *)
+module Message : sig
```

Comment:
I usually argue to write inner modules in full uppercase. It is a good convention to navigate in the code and make the difference between modules that have a source file and modules that are internal only.

Reply:
It is not meant to be internal however? It is exposed and users are expected to call functions from this module.

I can move it to a separate file, it represents an actual Zulip message and doesn't really have anything to do with "events", except that it is produced by the "events" endpoint.

Reply:
Internal in the sense that the module is within another module, the convention is to use full uppercase for such modules